Configuration/src/Steeltoe.Samples.Configuration/Program.cs

```diff
@@ -0,0 +1,54 @@
+using Steeltoe.Configuration.CloudFoundry;
+using Steeltoe.Configuration.ConfigServer;
+using Steeltoe.Configuration.Kubernetes.ServiceBinding;
+using Steeltoe.Configuration.Placeholder;
+using Steeltoe.Configuration.RandomValue;
+using Steeltoe.Management.Endpoint;
+using Steeltoe.Samples.Configuration.Models;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Steeltoe: Add Configuration providers.
+builder.Configuration.AddRandomValueSource();
+builder.Configuration.AddKubernetesServiceBindings();
+builder.Configuration.AddCloudFoundry();
+builder.Configuration.AddPlaceholderResolver();
+builder.AddConfigServer();
+
+// Steeltoe: Add actuator endpoints.
+builder.AddAllActuators();
+
+// Add services to the container.
+builder.Services.AddControllersWithViews();
+
+builder.Services.ConfigureCloudFoundryOptions(builder.Configuration);
```

Comment:
What comment can be added here to describe why someone would want to call this method? Does it do anything the sample needs?

Reply:
added this

> // Steeltoe: map VCAP_APPLICATION and VCAP_SERVICES to IOptions<CloudFoundryApplicationOptions> and IOptions<CloudFoundryServicesOptions>